src/typescript/frontend/src/components/selects/select/index.tsx

```diff
@@ -29,11 +29,11 @@ export const Select: React.FC<SelectProps> = ({
 
   return (
     <DropdownSelectWrapper onMouseEnter={replay} {...wrapperProps}>
-      <FlexGap gap="8px" ellipsis>
+      <FlexGap ref={targetRef} gap="8px" ellipsis>
         <Text className={"med-pixel-text"} color="darkGray">
           {"{"}
         </Text>
-        <Text ref={targetRef} className={"med-pixel-text"} {...placeholderProps} ellipsis>
+        <Text className={"med-pixel-text"} {...placeholderProps} ellipsis>
```

Comment:
```suggestion
        <Text className="med-pixel-text" {...placeholderProps} ellipsis>
```
nit, but raw string props are easier to parse visually with no braces imo. Feel free to leave as is